.github/workflows/build.yml

```diff
@@ -51,6 +51,9 @@ jobs:
     name: Test Suite
     runs-on: ${{ matrix.os }}
     steps:
+      - name: Install dependencies
```

Comment:
`if: matrix.os == "ubuntu-latest"`

Reply:
Exactly what I was looking for

Reply:
Note: need to add this dependency to `release.yml` too